src/app/components/show-user/show-user.component.html

```diff
@@ -1,4 +1,10 @@
-<div *ngFor="let user of store.users | async">
-  <h3>{{ user.attributes.fullName }}</h3>
-  <h4>{{ user.attributes.email }}</h4>
+<div *ngIf = "checkUserExist; else noUser">
```

Comment:
Variables should not be named like verb `checkUserExist`. We name functions like that. It should be something like `isUser`. Or better yet just check the length of the `users` array here. No need to have another function.

Reply:
the user array is initially empty, so using the length of the user array will trigger the else block in the html on initial loading or any subsequent reloading of the app.